cmd/scaffold.go

```diff
@@ -461,14 +461,17 @@ func (fnb *FlowNodeBuilder) initMetrics() {
 func (fnb *FlowNodeBuilder) initProfiler() {
 	// note: by default the Golang heap profiling rate is on and can be set even if the profiler is NOT enabled
 	runtime.MemProfileRate = fnb.BaseConfig.profilerMemProfileRate
+
 	if !fnb.BaseConfig.profilerEnabled {
```

Comment:
Don't we have to remove this line now? Otherwise the auto profiler will never get created during initialization, if we set the profiler to be false during startup